schemas/EGA.experiment.json

```diff
@@ -2,12 +2,331 @@
+              {
+                "title": "Assay type and subtype terms are from the sequencer CV list",
+                "properties": {
+                  "assay_type": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_type_by_sequencer.json" 
+                  },
+                  "assay_subtype": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_sequencer.json" 
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      },
+
+      "assayed_molecule_descriptor": {
+        "type": "object",
+        "title": "Descriptor of the assayed molecule",
+        "description": "Node containing information about the assayed molecule: the material entity (e.g. DNA) that was used to generate the data.",
+        "additionalProperties": false,
+        "required": ["assayed_molecule_type"],
+        "properties": {
+          "assayed_molecule_type": {
+            "type": "string",
+            "title": "Type of the assayed molecule of the experiment",
+            "description": "The assayed molecule (e.g. DNA), chosen from a list of controlled vocabulary (CV). If you cannot find your term in the CV list, please create an issue at our [metadata GitHub repository](https://github.com/EbiEga/ega-metadata-schema) proposing its addition.",
+            "enum": ["DNA", "RNA", "metabolie", "protein"],
+            "meta:enum": ["DNA__CHEBI:16991", "RNA__CHEBI:33697", "metabolite__EFO:0004727", "protein__CHEBI:36080"]
+          },
+          "assayed_molecule_subtype": {
+            "type": "string",
+            "title": "Subtype of the assayed molecule of the experiment",
+            "description": "Name of the assayed molecule subtype. We highly recommend the usage of names given to ontologized molecule subtypes, specially those at the [Experimental Factor Ontology (EFO)](https://www.ebi.ac.uk/ols/ontologies/efo).",
+            "examples": ["cDNA", "genomic DNA", "mitochondrial DNA", "messenger RNA", "ncRNA", "insulin", "serum albumin" ]
+          },
+          "assayed_molecule_subtype_curie": {
+            "type": "string",
+            "title": "CURIE of the assayed molecule subtype",
+            "description": "Ontology term in CURIE format (e.g. 'PR:000003918') of the assayed molecule subtype. Search for the ontologized term at the [Ontology Lookup Service (OLS)](https://www.ebi.ac.uk/ols/index).",
+            "examples": ["PR:000003918", "EFO:0008481", "EFO:0008479", "EFO:0008480", "CHEBI:33699"],
+            "allOf": [
+              {
+                "title": "General pattern of a CURIE",
+                "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/curie_general_pattern"
+              }
+            ]
+          }
+        }
+      },
+
```

Comment:
This is what I meant, @malloryfreeberg, with [my comment](https://github.com/EbiEga/ega-metadata-schema/pull/34#discussion_r820587615)